app/src/components/InputElement/InputElement.js

```diff
@@ -1,12 +1,12 @@
 import React from "react";
-import { StyledForm } from "./InputElement.styles";
+import { StyledForm, StyledButton, StyledInputField } from "./InputElement.styles";
 
 export const InputElement = () => {
   return (
     <StyledForm>
       <label for="url_input"></label>
-      <input id="url_input" type="text" />
-      <button>Shorten It!</button>
+      <StyledInputField id="url_input" type="text" placeholder="      Shorten a link here..."/>
```

Comment:
Why is there such a long empty space in the placeholder? Please remove it :D

Reply:
I wanted to move placeholder text to the right. I removed space and added padding-left in styles.